include/tester/db/Logger.hpp

```diff
@@ -34,11 +34,11 @@ class Logger
         {
             _document << "creation-time" << getDate();
             _open = true;
+            _data.clear();
```

Comment:
Have you tested this works with this line here?

Reply:
Yes, this is working, without this, the _data isn't getting clear, and just adding data to it.